addons/assignGear/functions/fnc_setContainerContentsFromConfig.sqf

```diff
@@ -0,0 +1,61 @@
+#include "script_component.hpp"
+/*
+ * Author: Bailey
```

Comment:
I'm not the author of this code, credit `PabstMirror`

Reply:
Gotcha, is that true for the other comment as well?

Reply:
Just this